mscore/pianoview.h

```diff
@@ -103,6 +103,7 @@ class PianoView : public QGraphicsView {
       bool inProgressUndoEvent;
       
       QList<PianoItem*> noteList;
+      char pitchHighlight[128];
```

Comment:
Not sure `char` is the right choice here

Reply:
I was trying to keep the memory footprint small.  What datatype would be better?

Reply:
Don't know, don't really see this type often... Anyway if `char` is indeed better, please specify `signed` or `unsigned`, see https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules#chars